src/server.c

```diff
@@ -602,10 +597,26 @@ int htNeedsResize(dict *dict) {
 /* If the percentage of used slots in the HT reaches HASHTABLE_MIN_FILL
  * we resize the hash table to save memory */
```

Comment:
I suppose this comment is no longer quite accurate, as this code is now more complex.

Reply:
It's still valid at an individual HT level though(dictionary per slot).

Reply:
I think there is two things. 
1. We are now overloading the term slot to mean two different things. We should probably update it to be bucket.
2. I think it's still worth mentioning this is now incremental. A single HT might be full but it will take some time to reclaim it.